verification.py

```diff
@@ -2264,6 +2305,37 @@ def run_xi_dirac_breakpoints(self):
                     L=10**6, model=msprime.DiracCoalescent(psi=psi, c=c),
                     growth_rate=0.05)
 
+    def run_xi_beta_recombinations(self):
+        basedir = "tmp__NOBACKUP__/xi_beta_recombinations"
+        for alpha in [1.1, 1.3, 1.6, 1.9]:
+            self.verify_recombination(
+                basedir, f"n=100_alpha={alpha}", sample_size=100, Ne=10**4, r=1e-8,
+                L=10**6, model=msprime.BetaCoalescent(alpha=alpha))
+
+
+    def run_xi_dirac_recombinations(self):
+        basedir = "tmp__NOBACKUP__/xi_dirac_recombinations"
+        for psi in [0.01, 0.1, 0.3, 0.6, 0.9]:
+            for c in [0.01, 1, 10, 100]:
+                self.verify_recombination(
+                    basedir, f"n=100_psi={psi}_c={c}",
+                    sample_size=100, Ne=10**4, r=1e-8,
+                    L=10**6, model=msprime.DiracCoalescent(psi=psi, c=c))
+
+    def run_Hudson_recombinations(self):
+        basedir = "tmp__NOBACKUP__/hudson_recombinations"
+        self.verify_recombination(
+            basedir, f"n=10_hudson",
+            sample_size=10, Ne=10**4, r=1e-9,
+            L=10**7, model="hudson")
+
+    def run_DTWF_recombinations(self):
```

Comment:
Can we remove this function please @TPPSellinger? We're not going to be able to run it, so no point in keeping it in there.

Reply:
Yes sure!  You can remove it. I was only using it because it was when I ran it that the error occured.